CitationStyleLanguagePlugin.inc.php

```diff
@@ -45,7 +48,8 @@ public function register($category, $path, $mainContextId = null) {
 		$success = parent::register($category, $path, $mainContextId);
 		if (!Config::getVar('general', 'installed') || defined('RUNNING_UPGRADE')) return $success;
 		if ($success && $this->getEnabled($mainContextId)) {
-			HookRegistry::register('ArticleHandler::view', array($this, 'getArticleTemplateData'));
+			HookRegistry::register('PreprintHandler::view', array($this, 'getSubmissionTemplateData'));
```

Comment:
Instead of overloading the `getSubmissionTemplateData` method, and checking `$this->applicationName == 'ops'`, can you use a new method, like `getPreprintTemplateData`? It will require a little bit of code be duplicated, but it will simplify each method and prevent the query URL in OPS from having irrelevant params like `issueId`.